client/src/components/Menu/CategoryDropMenuIcon.js

```diff
@@ -1,12 +1,13 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, { Component } from 'react';
 import './menu.css';
+import { withRouter } from 'react-router-dom';
+import { Route } from 'react-router-dom';
 
 function CategoryDropMenuIcon(props) {
     return (
         <div className="category-card">
             <div className="category-icon">
-                <Link to={props.href}><img src={props.src} alt={props.alt} /></Link>
+                <button conClick={() => {props.history.push('/categories/5a237266457b4c069453ec68') }} to={props.href}><img src={props.src} alt={props.alt} /></button>
```

Comment:
this is hardcoded to always go to the same category?

Reply:
I searched a lot but there was not clear explanation, I will carry on to find better way